packages/localize/test/date/getMonthNames.test.js

```diff
@@ -27,7 +27,7 @@ describe('getMonthNames', () => {
     //   s`Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec`,
     // );
     expect(getMonthNames({ locale: 'nl-NL', style: 'short' })).to.deep.equal(
```

Comment:
It seems this fails now differently on WebKit <> Firefox/Webkit. Maybe this can be commented out for now?